### DISCUSSION TITLE
Uri created by WC Dapp didn't work

### DISCUSSION THREAD

**Discussion by kevinamado** [WalletConnect/walletconnect-monorepo#1094] [Developer Support]
I'm stuck with the URI. 

I use : 
Dapp from https://github.com/WalletConnect/WalletConnectKotlinV2
Wallet from https://react-wallet.walletconnect.com/

To make some tests, I tried to connect the Dapp with a lot of wallet like Metamask, Steakwallet, Infinity, Unstoppable, KEYRING PRO, but no one respond to the deeplink and didn't display an error. So I copy the deeplink created by the app WC Dapp like this one for example:

wc:bdebc8b0ff3e0b78310e3dc382af729ab3f1d984497c572b4a800ec1e54737d8@2?relay-protocol=waku&symKey=ea043e7e58271714b8a029aacc87679cbee0170f7616229d0f70cb066d9aee32

and past it on the wallet at https://react-wallet.walletconnect.com/

And error show up and say :

Error: Cannot safe json parse value of type undefined

I don't understand why I can't connect the WC Dapp to no one Wallet. Can someone can help me ? :)

**Comment by chadyj** [comment]
There could be a compatibility issue.
Can you please try using https://react-wallet-beta.walletconnect.com and reporting if the problem still occurs?
